packages/host/app/components/operator-mode.gts

```diff
@@ -0,0 +1,98 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { Button } from '@cardstack/boxel-ui';
+import CardCatalogModal from '@cardstack/host/components/card-catalog-modal';
+import CreateCardModal from '@cardstack/host/components/create-card-modal';
+import { restartableTask } from 'ember-concurrency';
+import {
+  chooseCard,
+  catalogEntryRef,
+  createNewCard,
+  baseRealm,
+} from '@cardstack/runtime-common';
+import { CatalogEntry } from 'https://cardstack.com/base/catalog-entry';
+import type LoaderService from '../services/loader-service';
+import { service } from '@ember/service';
+import type * as CardAPI from 'https://cardstack.com/base/card-api';
+import type { ComponentLike } from '@glint/template';
+import { tracked } from '@glimmer/tracking';
+
+interface Signature {
+  Args: {
+    firstCardInStack: ComponentLike;
```

Comment:
Not a blocker, but I suspect that we may want to refactor this. It's unclear to me why the first card in a stack would be a special case and require a dedicated argument. I would imagine instead that the URL or other state persistence reflects the stack or stacks of cards and that this entire data structure flows down to the OperatorMode component and into the Stack components.

Reply:
Agree. Will create/adjust a ticket to reflect that idea.